zbsprimarystorage/zbsprimarystorage/zdaemon.py

```
@@ -0,0 +1,53 @@
+
+
+def prepare_pid_dir(path):
+    pdir = os.path.dirname(path)
+    if not os.path.isdir(pdir):
+        os.makedirs(pdir)
+
+
+def main():
+    usage = 'usage: python -c "from zbsprimarystorage import zdaemon; zdaemon.main()" start|stop|restart'
+    if len(sys.argv) != 2 or not sys.argv[1] in ['start', 'stop', 'restart']:
+        print
```

Ruff (0.12.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)